setup.py

```diff
@@ -25,6 +20,10 @@
     author_email="neuropoly@googlegroups.com",
     keywords="",
     packages=find_packages(exclude=["contrib", "docs", "tests"]),
-    install_requires=install_reqs,
-    package_dir={"shimmingtoolbox": "shimmingtoolbox"},
+    install_requires=[
+        #"numpy~=1.16.0",
```

Comment:
To be uncommented as soon as needed. I left this in as an example to get us started.